# When building swpaon, a script is run to determine
# details of the target system, and this generates header files with the
# correct defines. Unfortunatly this doesn't take account of cross
# compilation, so generate the file correctly here.

--- util-linux-2.11z.orig/mount/swapargs.h	1970-01-01 01:00:00.000000000 +0100
+++ util-linux-2.11z/mount/swapargs.h	2003-09-16 12:07:03.000000000 +0100
@@ -0,0 +1,3 @@
+#define SWAPON_HAS_TWO_ARGS
+#include <asm/page.h>
+#include <sys/swap.h>
